ts/Dashboards/EditMode/EditContextMenu.ts

```diff
@@ -47,21 +47,7 @@ class EditContextMenu extends Menu {
     /**
      * Default Context menu items.
      */
-    public static items = merge(Menu.items, {
-        editMode: {
-            id: 'editMode',
-            type: 'toggle',
-            getValue: function (item: MenuItem): boolean {
-                return item.menu.editMode.isActive();
-            },
-            langKey: 'editMode',
-            events: {
-                click: function (this: MenuItem): void {
-                    (this.menu as EditContextMenu).editMode.onEditModeToggle();
-                }
-            }
-        }
-    });
+    public static items = merge(Menu.items, {});
```

Comment:
Do we need merge here?

Reply:
`merge` create a copy, IMO it's safer to work on a fresh copy than accidentally modify an existing object.